experiments/hello-network-socket-ping/local_pong.c

```diff
@@ -0,0 +1,212 @@
+#include <errno.h>
+#include <stdio.h>
+#include <unistd.h>
+
+#include <sys/un.h>
+#include <sys/socket.h>
+
+#include "shared.h"
+#include "util.h"
+
+#define SIZE_RECEIVE_BUFFER 512
+
+const char* g_program_name = "Pong";
+
+// TODO
+// * write separate files for Unix domain sockets and internet domain sockets?
+
+// TODO consider rename to: prepare_for_bind_server
```

Comment:
Please address this TODO.

Reply:
Done in: https://github.com/arbiter0xf/neotetris/pull/8/commits/4a6e67db7dd739e0d57c804041f6c0642d806b7d